drivers/gpio/gpio_emul_sdl_bottom.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022, Basalte bv
+ * Copyright (c) 2023 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * "Bottom" of the SDL GPIO emulator.
+ * When built with the native_simulator this will be built in the runner context,
+ * that is, with the host C library, and with the host include paths.
+ */
+
+#ifndef DRIVERS_GPIO_GPIO_EMUL_SDL_BOTTOM_H
+#define DRIVERS_GPIO_GPIO_EMUL_SDL_BOTTOM_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Note: None of these are public interfaces. But internal to the SDL GPIO emulator */
+
+#define GPIOEMULSDL_SCANCODE_UNKNOWN 0
+
+struct gpio_sdl_data_bottom {
```

Comment:
just `gpio_sdl_data`? (the input one is not suffixed, just to keep them coherent)